### DISCUSSION TITLE
How to get the contents of a page?

### DISCUSSION THREAD

**Discussion by yosefbernal** [notion-dotnet/notion-sdk-net#350] [Q&A]
I've looked around but I see no `Content` property or `List<Block>` in the response of `client.Pages.RetrieveAsync(pid)`. Help!

**Comment by KoditkarVedant** [comment]
@yosefbernal Sorry for delayed response. It bit confusing from Notion side but you can use the `client.Blocks.RetrieveChildrenAsync(pageId)` pass `page_id` - that returns the children blocks inside a page. Hope this helps.
